File: proyect_rappi/python/Delivering.py
from User import User
from Deliver import Deliver
from Route import Route
from Payment import Payment
from Products import Products

class Delivering:
    id = int
    User = User("","","")
    product = Products("","","")
    deliver = Deliver("","")
    route = Route
    payment = Payment
    def __init__(self,user,product,deliver,payment):
        self.user = user
        self.product = product
        self.deliver = deliver
        self.payment = payment

File: proyect_rappi/Java/Delivering.java
class Delivering {
    Integer id;
    User user;
    Products product;
    Deliver deliver;
    Route route;
    Boolean completed;
    Payment payment;

    public Delivering(User user,Deliver deliver,Products product,Payment payment) {
    this.user = user;
    this.deliver = deliver;
    this.product = product;
    this.payment = payment;
    }
    void imp_deliver(){
        System.out.println("la entrega se hace: \n");
        System.out.println("usuario :"+user.name+" con el repartidor: "+deliver.name+ " producto: "+product.name+" brand: "+product.brand);
    }
    public void setAmount(int i) {
    }
}
